.github/helper/semgrep_rules/frappe_correctness.py

```diff
@@ -0,0 +1,28 @@
+import frappe
+from frappe import _, flt
+
+from frappe.model.document import Document
+
+
+def on_submit(self):
```

Comment:
You can specify `# noqa` to pass through sider check.

Reply:
Sure, will do

Reply:
I don't think DB CI failure is related to any of the changes introduced in this PR. Coveralls is failing for some other reason.